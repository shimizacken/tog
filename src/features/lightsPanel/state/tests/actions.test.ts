import {
  setFrontLightState,
  setBackLightState,
  setCabinLightState,
  setWagonsLightState,
} from "../actions";

describe("lights actions test suite", () => {
  it("should return setFrontLightState action", () => {
    expect(setFrontLightState).toMatchSnapshot();
  });

  it("should return setBackLightState", () => {
    expect(setBackLightState).toMatchSnapshot();
  });

  it("should return setCabinLightState action", () => {
    expect(setCabinLightState).toMatchSnapshot();
  });

  it("should return setWagonsLightState action", () => {
    expect(setWagonsLightState).toMatchSnapshot();
  });
});
