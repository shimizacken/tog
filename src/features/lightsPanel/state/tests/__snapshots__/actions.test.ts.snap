// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lights actions test suite should return setBackLightState 1`] = `
Object {
  "type": "SET_BACK_LIGHT_STATE",
}
`;

exports[`lights actions test suite should return setCabinLightState action 1`] = `
Object {
  "type": "SET_CABIN_LIGHT_STATE",
}
`;

exports[`lights actions test suite should return setFrontLightState action 1`] = `
Object {
  "type": "SET_FRONT_LIGHT_STATE",
}
`;

exports[`lights actions test suite should return setWagonsLightState action 1`] = `
Object {
  "type": "SET_WAGON_LIGHT_STATE",
}
`;
